src/app/services/story.service.ts

```diff
@@ -13,7 +13,17 @@ export class StoryService {
   constructor(private http: HttpClient) { }
 
   getStory(resource: String, uid: String): Observable<any> {
-    const url = `${this.apiUrl}story/${resource}/${uid}`;
+    let fallback = null;
+    let targetResource = resource.toLowerCase();
+    if (targetResource === 'advisory' || targetResource === 'kojibuild') {
+      // Try the container labels first and fallback to the general labels
+      fallback = targetResource;
+      targetResource = `container${targetResource}`;
```

Comment:
So consider this scenario, if the user requests for a KojiBuild (which is also a ContainerKojiBuild), this will query the node as a ContainerKojiBuild which in turn will place this build after FreshmakerEvent in the story path whereas the user will be expecting it after a DistGitCommit right. Is this situation handled? I'm sorry if I am missing something

Reply:
That is exactly right. If the user requests a build and it is a container build, it always be shown in the context of a container build (after a Freshmaker event). I think it's a viable approach but I'm open to other ideas.

Reply:
How about making two queries, one as a KojiBuild and other as a ContainerKojiBuild, and then we show the longer story?

Reply:
We talked about this out of band and decided this is a simple and viable approach. We will revisit this discussion if the requirements change.